content/en/penalties.json

```diff
@@ -0,0 +1,19 @@
+[
+    {"title": "Apologize"},
+    {"title": "Swear"},
+    {"title": "Snap your fingers"},
+    {"title": "Clap audibly"},
+    {"title": "Mispronounce a word"},
+    {"title": "Hum three notes"},
```

Comment:
I thought some of these were removed?

Reply:
Yeah I'm working my way through the new prompts at the moment. Will tackle these soon.